web/Spinner.qml

```diff
@@ -5,6 +5,8 @@ WebItem {
 	height: 80;
 	transform.rotate: 0;
 	property int squize: 0;
```

Comment:
can you rename squize to squeeze please? I know it's not yours, but it would be nice to fix it.